great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -820,11 +821,17 @@ def _persist_suite_with_store(
 
         return expectation_suite
 
-    def _save_project_config(self) -> None:
+    def _save_project_config(
+        self, fds_datasource: FluentDatasource | None = None
+    ) -> None:
```

Comment:
Isn't this signature a deviation from the parent class?

Reply:
I've added the optional parameter to all of them.
I didn't add types to all of them but mypy didn't have a problem with that. Still, I can make them all have the same annotations.